source/css/scss-common/objects/_buttons.scss

```diff
@@ -666,7 +666,7 @@
 
 // sass-lint:disable-block no-disallowed-properties
 @media (min-width: 992px) {
-  li & .globalNav-active {
+  li {
```

Comment:
tror denne kan være litt for generell

Reply:
Endret

Reply:
Fixed, går nå bare på nav .a-globalNav li